.eslintrc.js

```diff
@@ -3,20 +3,76 @@ module.exports = {
     browser: false,
     commonjs: true,
     es2020: true,
-    "jest/globals": true,
   },
   extends: [
     "eslint:recommended",
     "plugin:node/recommended",
-    "plugin:package-json/recommended",
+    "plugin:import/recommended",
   ],
-  plugins: ["package-json", "jest"],
+  plugins: ["node"],
   globals: {
     Atomics: "readonly",
     SharedArrayBuffer: "readonly",
   },
   parserOptions: {
     ecmaVersion: 2019,
   },
-  rules: {},
+  settings: {
+    node: {
+      allowModules: ["expect-puppeteer"],
+      resolvePaths: [__dirname],
+      tryExtensions: [".js", ".json", ".node"],
+    },
+  },
+  overrides: [
+    {
+      files: ["**/package.json"],
+      plugins: ["package-json"],
+      extends: "plugin:package-json/recommended",
+    },
+    {
+      files: ["**/*.test.js", "kumascript/tests/macros/utils.js"],
+      globals: {
+        page: true,
+      },
+      plugins: ["jest"],
+      extends: "plugin:jest/recommended",
+      rules: {
+        "jest/no-standalone-expect": [
+          "error",
+          {
+            additionalTestBlockFunctions: ["itMacro", "withDeveloping"],
+          },
+        ],
+        "no-unused-vars": 0,
+        "node/no-deprecated-api": 0,
+        "jest/valid-title": 0,
+        "no-useless-escape": 0,
+        "no-undef": 0,
+        "jest/expect-expect": 0,
+        "jest/no-conditional-expect": 0,
+      },
+    },
+    {
+      files: ["**/*.ts"],
```

Comment:
There are no `.ts` files outside of `client/` and that's ignored. No?

Reply:
Yeah, I set it up before ignoring the client folder. I think it's better to leave so it's easier to opt-back into that folder when there is time to look at the issues there

Reply:
A also forgot to add `.txs`, but that's something for later 😀

Reply:
Then let's delete it. Please. I much prefer that we let the existing configuration live where it currently lives and the day we rewrite the Node code to TypeScript, we'll tackle that then.